src/main/kotlin/us/ihmc/ci/IHMCCIPlugin.kt

```diff
@@ -218,9 +218,15 @@ class IHMCCIPlugin : Plugin<Project>
                // List tests to be run
                LogTools.quiet("Tests to be run:")
                testsToTagsMap.value.forEach { entry ->
-                  if ((category == "fast" && entry.value.isEmpty()) || entry.value.contains(category))
+                  val testName = entry.key;
+                  val tags = entry.value;
+
+                  val isInFast = tags.isEmpty() && category == "fast" || tags.contains("fast")
+                  val willRun = isInFast || tags.contains(category);
+
+                  if (willRun)
                   {
-                     LogTools.info(entry.key + " " + entry.value)
+                     LogTools.quiet("$testName $tags")
```

Comment:
We need to use quiet instead of info here since thats what we do above
`LogTools.quiet("Tests to be run:")`